openjdk.test.jlm/build.xml

```diff
@@ -0,0 +1,102 @@
+<?xml version="1.0"?>
+
+<!--
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+<project name="openjdk.test.jlm" default="build">
+
+	<!-- Set default for source_root.  -->
+	<property name="source_root" location="../"/>
+
+	<!-- Set default for STF location.  -->
+	<!-- Assumes source repositories are checked out under a common root and the repository
+		has been built in-situ - i.e.
+		stf repository: /home/userid/git/stf
+		openjdk-systemtest repository: /home/userid/git/openjdk-systemtest
+		If the stf repository has been cloned or built to a different location, the stf_root property
+		must be set on the command line - e.g. -Dstf_root=/my_stf_build_dir
+	-->
+	<property name="stf_root" location="${source_root}/../stf"/>
+
+	<!-- Import settings used by multiple projects.  -->
+	<import file="${source_root}/openjdk.build/include/top.xml"/>
+		
+	<!-- For modularity you need a directory above the package structure to hold the module.java file -->
+	<property name="module" value="test.jlm" />
+	<property name="src_dir" value="src/${module}" />
+	<property name="bin_dir" value="bin" />
+
+	<property name="jar_file" value="${bin_dir}/${module}.jar" />
+	
+	<!-- We need junit and stf to compile this project. -->
+	<path id="project.class.path">
+		<path refid="junit.class.path" />
+		<path refid="stf.class.path" />
+	</path>
+	
+	<!-- Projects which need to be built before this one. -->
+		<!-- dir must be set on the ant task otherwise the basedir property is not set to a new value in the subant task. -->
+		<target name="build-dependencies">
+			<!-- <ant antfile="${stf_root}/stf.build/build.xml" dir="${stf_root}/stf.build" inheritAll="false"/> -->
```

Comment:
Line is commented out, is it not needed?

Reply:
We don't need this `build-dependencies` target. This code needs to be cleaned up. Just discovered that, the same clean up needs to happen in other openjdk-systemtest projects too, where the same commented out code remains.

 We don't need to build stf as part of building each individual test project, as stf is built from openjdk.build prior to building the test projects.